app/app.go

```diff
@@ -26,6 +26,7 @@ import (
 // and its only purpose is to provide business logic to Server via its methods.
 type App struct {
 	ch *Channels
+	Id string
```

Comment:
for Golang abbreviations like ID and URL are generally going to be capital if not using a library 
```suggestion
	ID string
```